README.md

```diff
@@ -1,64 +1,104 @@
+- [About Helpless Kiwi](#about-helpless-kiwi)
+- [Getting Started](#getting-started)
+	- [Deployment](#deployment)
+	- [Development](#development)
+- [Contributing](#contributing)
+- [Partners](#partners)
+- [Maintainers](#maintainers)
+- [Contact](#contact)
+- [License](#license)
 
-| **Service** | **Master** | **Develop** |
-|-------------|------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------|
-| CI Status | ![CI](https://github.com/jasperweyne/helpless-kiwi/workflows/CI/badge.svg?branch=master) | ![CI](https://github.com/jasperweyne/helpless-kiwi/workflows/CI/badge.svg?branch=develop) |
 
-# Helpless Kiwi
-Helpless Kiwi is a project to manage members, create and manage activities, keep
-track of inventory and more. It has been written using the Symfony/Doctrine framework.
+## About Helpless Kiwi
+Helpless Kiwi is a (self-hosted) activity manager for (student) associations.
+It can help you manage your activities and the participation to them. Helpless
+Kiwi should give you insight into which activities do well, as well as allow
+your members a convenient and central way to view and sign up for them.
 
-## Need Help?
-If you've found a problem, or have a cool new feature in mind, post it in the
-[Issues](https://github.com/jasperweyne/helpless-kiwi/issues)! If you have a
-general question about Kiwi, come join us on [Discord](https://discord.gg/4HUmvEnXn8).
+## Getting started
+Regardless of if you're going to develop or wanting to deploy. Kiwi has some
+requirements and dependencies. All of which you can find
+[here](PREREQUISITES.md).
```

Comment:
These are not applicable to a Docker/Gitpod env, but are specified as such nonetheless currently. Moreover, given its short length, a user might glance over it easily and go straight to deployment/development. Therefore, a different structure could prove useful here

Reply:
I mean, technically it still has the dependencies. docker/gitpod just takes care of them for you. Not sure if I agree with you on it needing a different structure, I changed it a bit however. let me know what you think